engines/private/code.cpp

```diff
@@ -82,6 +82,7 @@ void SettingMaps::save(const char *name) {
 }
 
 void SettingMaps::load(const Common::String &name) {
+	//debug("loading: %s", name.c_str());
```

Comment:
I'm removing this, since it was redundant. It is implemented in the following lines: `debugC(1, kPrivateDebugCode, "loading setting %s", name.c_str());`